README.md

```diff
@@ -39,15 +30,15 @@ cd fpm
 Make sure that the development library of `gmp` is installed (e.g. `sudo apt install libgmp-dev` on Debian-derived Linux distributions)
```

Comment:
This line is no longer required; if installing manually, then the [manual installation](https://docs.haskellstack.org/en/stable/install_and_upgrade/#manual-download_2) instructions list which prerequisites to install, including `libgmp-dev`.

Reply:
Done!